### PR TITLE
[tests-only] Consistent interaction names

### PR DESCRIPTION
**Pull request by saw-jan**
This PR makes pact interaction names more consistent
All interaction names follow the same pattern i.e. `.uponReceiving(a <METHOD> request ...)`

closes https://github.com/owncloud/owncloud-sdk/issues/723

### CODE REVIEW

File: tests/userTest.js [hunk 1]
```diff
@@ -549,7 +549,7 @@ describe('Main: Currently testing user management,', function () {
     await getCurrentUserInformationInteraction(provider)
     await getUsersInteraction(
       provider,
-      'search for a user that exists',
+      'check user existence with existing user',
```
Comment (Talank) [inline tests/userTest.js:552]:
```suggestion
      'check user existence with existing users',
```

File: tests/userTest.js [hunk 2]
```diff
@@ -575,7 +575,7 @@ describe('Main: Currently testing user management,', function () {
     await getCurrentUserInformationInteraction(provider)
     await getUsersInteraction(
       provider,
-      'search for a user that doesn\'t exists',
+      'check user existence with non-existing user',
```
Comment (Talank) [inline tests/userTest.js:578]:
```suggestion
      'check user existence with non-existing users',
```

File: tests/userTest.js (hunk 1 above)
Comment (saw-jan) [reply tests/userTest.js:552]: `users` would be incorrect as it is more like checking user existence using one non-existing user in the request

File: tests/userTest.js (hunk 2 above)
Comment (saw-jan) [reply tests/userTest.js:578]: `users` would be incorrect as it is more like checking user existence using one non-existing user in the request

**Comment by Talank** [comment]
LGTM

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

File: tests/filesTest.js [hunk 3]
```diff
@@ -1337,12 +1336,12 @@ describe('Main: Currently testing files management,', function () {
...
         })
       await provider
-        .uponReceiving('PROPFIND file info, fileId')
+        .uponReceiving('a PROPFIND request to file info, fileId')
         .withRequest({
           method: 'PROPFIND',
           path: webdavPath(`${testFolder}/${testFile}`),
           headers: validAuthHeaders,
-          body: new XmlBuilder('1.0', '', 'd:profind').build(dPropfind => {
+          body: new XmlBuilder('1.0', '', 'd:propfind').build(dPropfind => {
```
Comment (individual-it) [inline tests/filesTest.js:1344]: good catch, but how did it work before?

File: tests/pactHelper.js [hunk 4]
```diff
@@ -342,7 +342,7 @@ const getUserInformationAsAdminInteraction = function (provider) {
 }
 
 const getCapabilitiesWithInvalidAuthInteraction = function (provider, username = config.adminUsername, password = config.invalidPassword) {
-  return provider.uponReceiving(`a capabilities GET request with invalid authentication by ${username} with ${password}`)
+  return provider.uponReceiving(`a GET request to get capabilities with invalid auth by '${username}' with '${password}'`)
```
Comment (individual-it) [inline tests/pactHelper.js:345]: should be `as user ..` if we follow the same structure like in the other requests

**Review by individual-it — CHANGES_REQUESTED** [review]
just one small change request
and for discussion: should we put `as ...` into every interaction name?

File: tests/filesTest.js (hunk 3 above)
Comment (saw-jan) [reply tests/filesTest.js:1344]: this test is skipped

File: tests/pactHelper.js (hunk 4 above)
Comment (saw-jan) [reply tests/pactHelper.js]: the decision was made to use `as <username>, a <METHOD> request ...`

**Review by individual-it — APPROVED** [review]

**Pull request merged**